src/Button/Button.react.js

```diff
@@ -89,18 +91,19 @@ function getStyles(props, context, state) {
         container: [
             button.container,
             !raised && buttonFlat.container,
-            disabled && buttonDisabled.container,
             raised && buttonRaised.container,
             local.container,
             props.style.container,
+            (disabled && raised) && buttonRaisedDisabled.container,
```

Comment:
Even though I would not use it myself, I'm sure there will be some developer that will use it thus, create a bug for that developer.  It's open source afther all 🛠.